include/ck/tensor_operation/gpu/device/device_generic_elementwise.hpp

```diff
@@ -0,0 +1,301 @@
+// SPDX-License-Identifier: MIT
```

Comment:
generic has no meaning, every op in CK is supposed to be  generic

rename to device_elementwise.cpp